src/act.c

```diff
@@ -2182,12 +2182,12 @@ void save_file_w( )
 	sprintf(path_work, "/ram/work.sav");
 	sprintf(path_item, "/ram/item_wk.sav");
 	
-#elif defined(_TINSPIRE)
-	sprintf(path_config, "./save/config.tns");

```

Comment:
The .tns extension (as well as the "./") is required for the TI-Nspire port to work properly because the operating system wants every file to have that extension (or else it will refuse to transfer it to the calc).